lib/src/multi_sig/multi_sig_builder.spec.ts

```diff
@@ -0,0 +1,209 @@
+import 'mocha';
+import { expect } from 'chai';
+import { NetworkEnum } from '../network';
+import { KeyPair } from '../key_pair';
+import { MultiSigBuilder } from '.';
+
+const network = NetworkEnum.Devnet;
+
+describe('MultiSigBuilder', () => {
+    const buyerKeyPair = KeyPair.generateRandom().toObject();
+    const sellerKeyPair = KeyPair.generateRandom().toObject();
+    let buyerBuilder: any;
+    let sellerBuilder: any;
+
+    describe('createMultiSigAddress', () => {
```

Comment:
Instead of using describe for consecutive steps, consider using
https://www.npmjs.com/package/mocha-steps

Reply:
Sure, I will try this out

Reply:
The reason not to use `describe` here is that when some of the previous steps fail, it will cause all the subsequent `describe` to fail. With it the test will stop sat where it first has problem.

Reply:
But it seems that the `mocha-steps` cannot do nested steps like

Reply:
I have verified the behaviour in nested `step`. I think the reason is `step` is replacing `it` but not `describe`, similar behaviour can be found if you have nested-it.

Perhaps you will have to flatten your test to use it. Or keep it in the current way.

Either way, another idea in my mind is all these tests are dependent on previous tests, which is undesired in unit test.`mocha-steps` serves as a way to 1) do early-failure of the whole suite and 2) tell that tests are in sequence, which is not immediately revealed to the reader when they first read the step.

Reply:
Resolved